WordPressUI.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
   s.name          = 'WordPressUI'
-  s.version       = '1.13.0'
+  s.version       = '1.13.1'
```

Comment:
Similar as above, this fix should be tagged `1.13.1-beta`.

Reply:
If it's convenient for the WooCommerce release schedule, we can bypass the beta step and ship a new stable version straightaway.

Let me know.

Reply:
I don't have a strong opinion about this, so I'll leave the question to @pmusolino.

Reply:
It would be beneficial to circumvent the beta phase, as the absence of this particular fix results in an app crash. @mokagio 🙌